ComponentTextKitApplicationTests/CKTextComponentTests.mm

```diff
@@ -384,4 +384,27 @@ - (void)testComplexLigatures
   CKSnapshotVerifyComponent(c, kUnrestrictedSize, @"");
 }
 
+- (void)testTruncationStringLargerThanLastLine
```

Comment:
Can you rename this test to indicate what *should* happen when the truncation string is larger than the last line?